### ISSUE TITLE
design folder should be split out from main repo

### ISSUE THREAD

**Issue by vise890** [scmbreeze/scm_breeze#220]
The design directory problem seems orthogonal to the git shortcuts that are the main functionality. There are also other (perhaps better?) solution to handle binary files in git (e.g. [`git-annex`](https://github.com/joeyh/git-annex).

**Comment by vise890** [comment]
it also requires the `~/.scmbrc` file in the home dir, which for non users is just clutter.

**Comment by jeffbyrnes** [comment]
@vise890 I’d be in favor of just dropping the feature.

**Comment by vise890** [comment]
@jeffbyrnes I'd love to do that as well. But how do we do about doing these sort of things. Do we need to maintain backwards compatibility? Is there a way to figure out how many people are actually using the feature?

**Comment by jeffbyrnes** [comment]
@vise890 no way to determine who’s using the feature, short of just removing it & seeing who complains, AFAIK.

I think it’s safe to put a big notice at the top of the README, and go ahead & rip it out. Folks can either make some noise for its return, or fork this repo, if they wish to keep it.

**Comment by ghthor** [comment]
I vote rip it out. We can tag the last version that includes it and link to that from the readme.

**Comment by vise890** [comment]
OK then. Is there any other stuff se want to deprecate? Ive never used the project index stuff...

**Comment by jeffbyrnes** [comment]
I used to use it, but nowadays, just use Bash’s built-in `$CDPATH`, and give it all the paths that have my projects.

**Comment by vise890** [comment]
so rip it all out ? what about just ripping out the git and numbering stuff into its own thing and adding it here as a sub-module? that way we don't need to worry about angering anyone and we can just bump the submodule version from time to time here

**Comment by ghthor** [comment]
Yeah, that could work. Calling the git+numbering repo scmgust or something?

**Comment by jeffbyrnes** [comment]
Ehh, I’d argue that the git + numbering stuff are the main feature here. If somebody is automatically upgrading their clone of SCM Breeze, they’ll come here & complain, it’s true. But I’d rather that, and we politely encourage them to fork an older commit to keep it, than confuse things with another repo/project.

**Comment by vise890** [comment]
right. so i think the 3 of us agree; git+numbering is the main thing we use. The two options are:

1. strip `scm_breeze` down to it
2. split it out into other project

@ghthor should we ask the original author for their input?

**Comment by ndbroadbent** [comment]
Hi, i'm happy to just remove it, I don't use it any more either :)

**Comment by vise890** [comment]
ok so to recap, we're happy to remove the "design folder" and "repo index" stuff from the main repo, yes?

**Comment by jeffbyrnes** [comment]
In case it was unclear from my reaction, I’m all-in on removing the design folder & repo index features.

**Comment by jeffbyrnes** [comment]
@ndbroadbent @ghthor with the recent activity here, thought I'd bump this & see if there's any appetite to remove these unused features.

**Comment by ghthor** [comment]
I do really want to do this, I'm just not sure when I'll find the time. I have some upcoming flights though, so unless I have other pressing things I'll try to take a look at this at least.

**Comment by jeffbyrnes** [comment]
@ghthor all good! Figured of all the old issues hanging out here, this one had the most merit & potential to both simplify & improve the project.

**Comment by jeffbyrnes** [comment]
@ghthor @ndbroadbent I’d be happy to help administer the repo, clean up old issues, etc., if you want a hand?

**Comment by ghthor** [comment]
🤔

**Comment by ndbroadbent** [comment]
Yes still in favor of removing. @jeffbyrnes Thanks that would be great!
